apps/docs/src/pages/forms/Switch.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <div>
+    <ForgePageHeader title="Switch" />
+    <p>
+      Further documentation and examples can be found in the
+      <a class="link" target="_blank" href="https://primevue.org/inputswitch/"><strong>PrimeVue documentation</strong></a>
+      .
+    </p>
+    <Playground :options="options" :code="code" :config="config" @reset="reset">
+      <template #component>
+        <component :is="InputSwitch" v-bind="options" v-model="value" />
+      </template>
+    </Playground>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { ForgePageHeader } from "@3squared/forge-ui-3";
+import InputSwitch from "primevue/inputswitch";
+import { Playground, usePlayground } from "@3squared/forge-playground-3";
+import { computed, ref } from "vue";
+
+const { options, propVals, config, reset } = usePlayground({
+  trueValue: "true",
+  falseValue: "false",
+  disabled: false,
+  readonly: false,
```

Comment:
Is readonly an accessibility thing? Just wondering the difference between this and disabled?

Reply:
So readonly just doesn't allow the disabled state (grey it out). It should just stop the user from changing the state of the switch without changing its appearance